etl/main.py

```diff
@@ -18,9 +16,18 @@
 for p in players:
     p['_id'] = p.pop('id')
 
+
+
 # upload to mongodb client (replace <password> with actual password)
-client = pymongo.MongoClient('mongodb+srv://steinar:<password>@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
-db = client.raw
+client = pymongo.MongoClient('mongodb+srv://' + config.username + ':' + config.password + '@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
```

Comment:
The client now requires config.py to access the username and password. #3

Reply:
Yes, this is a nice option.
You could also write what's called an 'f-string', which would look like this:
```python
client = pymongo.MongoClient(f'mongodb+srv://{config.username}:{config.password}@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')

Reply:
That's good to know!